t/lib/t/MusicBrainz/Server/Controller/WS/2/DefaultFormat.pm

```diff
@@ -0,0 +1,91 @@
+package t::MusicBrainz::Server::Controller::WS::2::DefaultFormat;
+
+use JSON qw( decode_json encode_json );
+use Test::Deep qw( cmp_deeply );
+use Test::JSON import => [qw( is_valid_json is_json )];
+use Test::More;
+use Test::Routine;
+use Test::XML::SemanticCompare;
+
+with 't::Mechanize', 't::Context';
+
+test 'MBS-9258: deduction of fmt from Accept header' => sub {
```

Comment:
Actually, there's at least one indexed search test in t/lib/t/MusicBrainz/Server/Controller/Search/Indexed.pm